src/lv_core/lv_refr.c

```diff
@@ -741,12 +741,80 @@ static void lv_refr_obj(lv_obj_t * obj, const lv_area_t * mask_ori_p)
     }
 }
 
+/**
+ * Rotate the VDB to the display's native orientation.
+ */
+static lv_color_t *lv_refr_vdb_rotate_pre(lv_disp_drv_t *drv, lv_area_t *area, lv_color_t *color_p) {
+    if(lv_disp_is_true_double_buf(disp_refr) && drv->sw_rotate) {
+        LV_LOG_ERROR("lv_refr_vdb_rotate_pre: cannot rotate a true double-buffered display!");
```

Comment:
`LV_LOG_ERROR` already adds the function name. See [here](https://github.com/lvgl/lvgl/blob/dfc6719479bab93657e981b6c82570003446071b/src/lv_misc/lv_log.h#L98).

(The function name is still there some old logs tough)